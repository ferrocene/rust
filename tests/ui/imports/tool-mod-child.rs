//@ revisions: edition2015 edition2018
//@ [edition2015] edition: 2015
//@ [edition2018] edition: 2018..

use clippy::a; //~ ERROR unresolved import `clippy`
use clippy::a::b; //[edition2015]~ ERROR failed to resolve: use of unresolved module or unlinked crate `clippy`
                  //[edition2018]~^ ERROR failed to resolve: `clippy` is a tool module, not a module

use rustdoc::a; //~ ERROR unresolved import `rustdoc`
use rustdoc::a::b; //~ ERROR failed to resolve: use of unresolved module or unlinked crate `rustdoc`

fn main() {}
