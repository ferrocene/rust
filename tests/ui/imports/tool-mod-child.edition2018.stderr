error[E0433]: failed to resolve: `clippy` is a tool module, not a module
  --> $DIR/tool-mod-child.rs:6:5
   |
LL | use clippy::a::b;
   |     ^^^^^^ `clippy` is a tool module, not a module

error[E0432]: unresolved import `clippy`
  --> $DIR/tool-mod-child.rs:5:5
   |
LL | use clippy::a;
   |     ^^^^^^ `clippy` is a tool module, not a module

error[E0433]: failed to resolve: use of unresolved module or unlinked crate `rustdoc`
  --> $DIR/tool-mod-child.rs:10:5
   |
LL | use rustdoc::a::b;
   |     ^^^^^^^ use of unresolved module or unlinked crate `rustdoc`
   |
   = help: you might be missing a crate named `rustdoc`

error[E0432]: unresolved import `rustdoc`
  --> $DIR/tool-mod-child.rs:9:5
   |
LL | use rustdoc::a;
   |     ^^^^^^^ use of unresolved module or unlinked crate `rustdoc`
   |
   = help: you might be missing a crate named `rustdoc`

error: aborting due to 4 previous errors

Some errors have detailed explanations: E0432, E0433.
For more information about an error, try `rustc --explain E0432`.
